src/posts/2022-10-18-kubecon-talks/index.md

```diff
@@ -0,0 +1,111 @@
+---
+path: '/blog/2022/10/17/cilium-kubecon-na-talks'
+date: '2022-10-17T17:00:00.000Z'
+title: 'Cilium Talks at KubeCon NA 2022'
+isPopular: true
+isFeatured: false
+ogImage: ciliumkubeconna.jpg
+ogSummary: "Find all the Cilium talk at KubeCon NA"
+categories:
+  - Community
+tags:
+  - Cilium
+  - KubeCon
+---
+
+![Cilium Talks at KubeCon](ciliumkubeconna.jpg)
+
+*October 17th, 2022*  
+*Author: Bill Mulligan, Isovalent*
+
+KubeCon + CloudNativeCon is already back around upon us and Cilium is represented in full force! There are 12 talks, workshops, and meetings that cover technical talks from Cilium contributors, tutorials to get people started, and end user stories from people putting Cilium into production. You will be able to catch presentations on everything from Cilium NetworkPolicies to Cilium for telco workloads to a tour of the Cilium for security. Let’s dive into each of the talks below.
+
+## Cloud Native eBPF Day
+
+[Keynote: eBPF - Everything You Need to Know in 5 Minutes - Thomas Graf, CTO, Isovalent](https://sched.co/1Auye)
+ 
+ Monday, October 24 • 9:15am - 9:20am
+
+eBPF has become the key technology for infrastructure software. This session tells you everything you need to know about eBPF in 5 minutes. Why eBPF matters and why it exists. What it can do. What it can’t do. Who uses it for what. And finally, what the future holds.
```

Comment:
```suggestion
eBPF has become the key technology for infrastructure software. This session tells you everything you need to know about eBPF in 5 minutes. Why eBPF matters and why it exists. What it can do. What it can’t do. Who uses it for what. And finally, what the future holds.

```